### DISCUSSION TITLE
java.lang.StackOverflowError: Unexpected error Bug

### DISCUSSION THREAD

**Discussion by SANS10808** [MoreMcmeta/core#92] [Help Wanted]
Whenever I open any world (Singleplayer and Multiplayer) it loads normally but then gives me java.lang.StackOverflowError: Unexpected error right when its done loading. Im using the latest versions of the Emissive mod and MoreMcMeta with Fabric and Sodium (and indium). it could be a compatibility issue with some of my other mods but I'm not sure. And when I remove the mods the game works normally.

**Comment by soir20** [comment]
Please provide the full stack trace and the version of the game you're running on--there's not enough info here for me to investigate this. I have tested the latest 1.21 version of both mods with Sodium and Indium, so it's likely a compatibility issue with another mod.

**Reply by SANS10808** [reply]
Not sure what a full stack trace is but I did find "Stacktrace" in the crash report, hope its what you were looking for. As for the game version, I'm using 1.21. If you need more info just let me know

Stacktrace:
	at net.minecraft.class_310.method_1587(class_310.java:2501)
	at net.minecraft.class_310.method_54580(class_310.java:949)
	at net.minecraft.class_310.method_1514(class_310.java:905)
	at net.minecraft.client.main.Main.main(Main.java:256)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

**Reply by soir20** [reply]
Could please you provide the full log file and/or crash report?

(What you provided is correct but I think there's still some of it missing--the crash report/logs will contain the full one.)

**Reply by SANS10808** [reply]
Here is the full crash report: https://pastebin.com/CfMkkhN7

**Reply by soir20** [reply]
Thanks for providing the crash report--it's a compatibility issue with Iris. This should hopefully be an easy fix.

**Reply by soir20** [reply]
MoreMcmeta emissive plugin version 2.0.5 has been released to fix this issue. This version also fixes an issue with block entity emissive overlays with Sodium. Thanks for reporting; please open a new issue for any other problems you encounter in the new version.
